chef/cookbooks/provisioner/recipes/base.rb

```diff
@@ -287,6 +287,20 @@
   end
 end
 
+# On SUSE: install relevant microcode packages based on the 1st CPU of the node
+if node[:platform_family] == "suse"
+  microcode_package = ""
+  if node[:cpu]
+    case node[:cpu]["0"][:model_name]
+    when /^Intel\(R\)/
+      microcode_package = "ucode-intel"
+    when /^AuthenticAMD/
+      microcode_package = "ucode-amd"
+    end
+  end
+  package microcode_package unless microcode_package.empty?
+end
```

Comment:
Should we really do it here? Why not in the autoyast profile?

Reply:
I've got a test branch locally that adds the installation into the autoyast template instead of this block, but that will not install the relevant ucode package on the admin node, which isn't installed via the provisioner barclamp, but it does get applied. If we think this is okay, then great, I'll push that up.

Reply:
The admin node is not our responsability, so I think it's okay.

Reply:
(but to be clear: not a strong feeling on my side)